subhub/api/webhooks/stripeWebhookPipeline.py

```diff
@@ -0,0 +1,52 @@
+from subhub.api.webhooks.stripeChargeCapturedProcessor import StripeChargeCapturedProcessor
+from subhub.api.webhooks.stripeChargeDisputeClosedProcessor import StripeChargeDisputeClosedProcessor
+from subhub.api.webhooks.stripeChargeDisputeCreatedProcessor import StripeChargeDisputeCreatedProcessor
+from subhub.api.webhooks.stripeChargeDisputeFundsReinstatedProcessor import StripeChargeDisputeFundsReinstatedProcessor
+from subhub.api.webhooks.stripeChargeDisputeFundsWithdrawnProcessor import StripeChargeDisputeFundsWithdrawnProcessor
+from subhub.api.webhooks.stripeChargeDisputeUpdatedProcessor import StripeChargeDisputeUpdatedProcessor
+from subhub.api.webhooks.stripeChargeExpiredProcessor import StripeChargeExpiredProcessor
+from subhub.api.webhooks.stripeChargeFailedProcessor import StripeChargeFailedProcessor
+from subhub.api.webhooks.stripeChargePendingProcessor import StripeChargePendingProcessor
+from subhub.api.webhooks.stripeChargeRefundUpdatedProcessor import StripeChargeRefundUpdatedProcessor
+from subhub.api.webhooks.stripeChargeRefundedProcessor import StripeChargeRefundedProcessor
+from subhub.api.webhooks.stripeChargeSucceededProcessor import StripeChargeSucceededProcessor
+from subhub.api.webhooks.stripeChargeUpdatedProcessor import StripeChargeUpdatedProcessor
+
+
+class StripeWebhookPipeline :
+
+    def __init__(self, payload):
+        assert isinstance(payload, object)
+        self.payload = payload
+
+    def run(self):
+        type = self.payload["type"]
```

Comment:
type is a keyboard in python that you have obfuscated with this assignment
https://stackoverflow.com/a/2418007